src/Core/Resolvers/SqlMutationEngine.cs

```diff
@@ -253,13 +253,13 @@ public class SqlMutationEngine : IMutationEngine
                     {
                         using (JsonDocument jsonDocument = JsonDocument.Parse(resultArray.ToJsonString()))
                         {
-                            return new CreatedResult(location: context.EntityName, OkMutationResponse(jsonDocument.RootElement.Clone()).Value);
+                            return new CreatedResult(location: string.Empty, OkMutationResponse(jsonDocument.RootElement.Clone()).Value);
```

Comment:
For SP, the Location is of the form `../api/<SP-Entity-Name>`

Since the baseURL already has the `Entity Name`, setting the entity name again in location results in incorrect Location header value --> `../api/CountBooks/CountBooks`. To prevent this, location needs to be set as an empty string.